reactNativeProject/src/screens/Main/components/Item/styles.js

```diff
@@ -1,21 +1,26 @@
 import { StyleSheet } from "react-native";
+import colors from "../../../../constants/colors";
+import variables from "../../../../constants/variables";
```

Comment:
Don´t move the variables to another file. You will only use that height and width here, so declare them on this file.
You don´t need to put them inside an object, just declare the `const`s here